mesheryctl/internal/cli/root/app/offboard.go

```diff
@@ -40,6 +42,36 @@ mesheryctl app offboard -f [filepath]
 			return errors.Wrap(err, "error processing config")
 		}
 
+		app := ""
```

Comment:
I see this particular code block getting repeated multiple times, can this be abstracted into a single function?
utils.ValidId(configtype string) (string, error)?